src/app/components/core/developer/additional-services/additional-services-utils.service.ts

```diff
@@ -27,7 +27,7 @@ export class AdditionalServicesUtils {
               // enabling service takes time to get the updated state from API
               setTimeout(() => {
                 this.router.navigate(['/developer/options/additional-services']);             
-              }, 2000);
+              }, 3000);
```

Comment:
As above

Reply:
Reason is already mentioned there

Reply:
How we can ensure that it will be available 3000 ms as well?

Reply:
I've tested it multiple times with this timestamp and it always passed but If you have better idea then let me know how can we ensure about the exact time?

Reply:
Think about the low end device like RPi, Have you tested on that? We should look for better observability and see how to trigger the event when previous call in the stack completed to notify for next action.